tools/mypy_wrapper.py

```diff
@@ -18,13 +18,13 @@
 - https://github.com/pytorch/pytorch/wiki/Lint-as-you-type
 """
 
-import fnmatch
 import re
 import sys
 from configparser import ConfigParser
 from itertools import chain
 from pathlib import Path, PurePath, PurePosixPath
 from typing import List, Set
+from wcmatch.glob import globmatch  # type: ignore[import]
```

Comment:
why do you need "# type: ignore[import]"?
Also, as `wcmatch` is not a standard package, it would be good to have a fallback to fnmatch
```suggestion
try:
    from wcmatch.glob import globmatch  as _fnmatch# type: ignore[import]
except ImportError:
  print("wcmatch not found, failing back to fnmatch")
  from fnmatch import fnmatchcase as _fnmatch
```

Reply:
The `# type: ignore[import]` is necessary to please `mypy` because `wcmatch` doesn't have type hints.

I'd rather not use a fallback to `fnmatch`, since the behavior is different (that's the entire point of this PR); if you'd rather avoid using a third-party package here, I could instead just write my own function for doing the matching, which would also avoid having to repeatedly call an external matching function in a loop in `mypy_wrapper.glob`.